### DISCUSSION TITLE
Feature Request: Text Highlight Feature

### DISCUSSION THREAD

**Discussion by volt90** [excalidraw/excalidraw#8278] [General]
How I like to use excalidraw has to do with highlighting certain texts within excalidraw. And I know that I can achieve the same effect by drawing a filled shape to attach to the back of the text, however, I think it would be much appreciated if I can click a button to go from plain text to text with highlighted background(kind of like how it is in Instagram). 

Thanks!

**Comment by Mrazator** [comment]
It's in our backlog, let's track it here #4323
